flanker/mime/message/headers/wrappers.py

```diff
@@ -72,7 +72,7 @@ def is_headers_container(self):
             self.is_disposition_notification()
 
     def is_rfc_headers(self):
-        return self == 'text/rfc822-headers'
+        return self == 'text/rfc822-headers' or 'message/rfc822-headers'
```

Comment:
should this not be `self == 'text/rfc822-headers' or self == 'message/rfc822-headers'`? this method is supposed to return a bool but instead, you are trying to return the `'message/rfc822-headers'` string when `self == 'text/rfc822-headers'` condition is not met 🤔